src/mechanics/css/actor.css

```diff
@@ -0,0 +1,3 @@
+.iron-vault-mechanics .actor {
```

Comment:
Oh this needs fixing once it’s time to style. Just gotta make sure not to merge into the mechanics node styling is done